htdocs/comm/propal/class/propal.class.php

```diff
@@ -1302,6 +1302,8 @@ function createFromClone($socid=0)
 
 		if (! $error)
 		{
+			// force load new object before launch hook 
```

Comment:
A test at begin of clone method to be sure caller has made the fetch would be better i think

Reply:
n fact the call is done just after the create of the clone and after the error test of the create; interest and push the clone into hook, otherwise some properties are not up to date. And the original object being passed by the param objFrom, only the $ object is not correct;
This is true on all calls to the createFromClone method; And it's particularly blocking on the property lines